horovod/common/operations.cc

```diff
@@ -210,13 +236,13 @@ bool IncrementTensorCount(std::unique_ptr<MessageTable>& message_table,
 // valid (for example, contained mismatched shapes or types).
 //
 // Constructing the Response, thus, requires a whole lot of error checking.
-Response ConstructResponse(std::unique_ptr<MessageTable>& message_table,
+Response ConstructResponse(std::unique_ptr<MessageTable> &message_table,
                            std::string name) {
   bool error = false;
   auto it = message_table->find(name);
   assert(it != message_table->end());
 
-  std::vector<Request>& requests = std::get<0>(it->second);
+  std::vector<Request> &requests = std::get<0>(it->second);
```

Comment:
Can you format using `.clang-format` file that's in the root of the repo?  It should  pin &s to the left.

Reply:
Fixed. Thank you for mentioning!